src/modules/date.js

```diff
@@ -66,7 +67,7 @@ const minicharts_d3fns_date = (appRegistry) => {
 
   // set up tooltips
   const tip = d3.tip()
-    .attr('class', 'd3-tip')
+    .attr('class', 'd3-tip d3-tip-date')
```

Comment:
The extra classes were just helpful for debugging, happy to remove them if that’s better